README.md

```diff
@@ -1,6 +1,6 @@
 # Gazebo for MAVLink SITL and HITL
 
-[![Build Status](https://travis-ci.org/PX4/sitl_gazebo.svg?branch=master)](https://travis-ci.org/PX4/sitl_gazebo)
+![Build Status](https://github.com/PX4/sitl_gazebo/workflows/Build%20Tests/badge.svg) ![MacOS Build Tests](https://github.com/PX4/sitl_gazebo/workflows/MacOS%20Build%20Tests/badge.svg) 
```

Comment:
You can put `![Build Status](https://github.com/PX4/sitl_gazebo/workflows/Build%20Tests/badge.svg)` inside `[ ]` and add `( )` with the link to the Actions pipeline report - otherwise this is just an image and not that useful. Check the Travis example that you are replacing.

Reply:
And the same goes with the MacOS badge.

Reply:
Done!